From a4ced833c7c6080d1e235385960f997402f5d961 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Wed, 6 Apr 2016 17:53:44 +0300
Subject: [PATCH 1010/1240] fix: hw_info: disable HW info feature for non
 development board

- issue found when disable devel board config the U-Boot will not compile

Change-Id: Iabe5e64186c83a8be105caa5b7d6cf1b75d67416
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28836
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 common/mvebu/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/mvebu/Kconfig b/common/mvebu/Kconfig
index 4b907e2..6b4cdba 100644
--- a/common/mvebu/Kconfig
+++ b/common/mvebu/Kconfig
@@ -73,6 +73,7 @@ config CMD_MVEBU_PHY_INDIRECT
 
 config CMD_MVEBU_HW_INFO
 	bool "hw_info"
+	depends on DEVEL_BOARD
 	select BOARD_CONFIG_EEPROM
 	default n
 	help
-- 
1.9.1

